.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
why do we need this?

Reply:
For some reason I've always had issues with babel-eslint. Dan tried to help me figure it out, but the only way it would work would to be to add this line.

Reply:
try `npm uninstall -g babel-eslint` then doing steps 1, 2, and 4